libs/s25main/GamePlayer.cpp

```diff
@@ -324,8 +324,15 @@ void GamePlayer::Deserialize(SerializedGameData& sgd)
     for(uint8_t& militarySetting : militarySettings_)
         militarySetting = sgd.PopUnsignedChar();
 
+#if defined(__GNUC__) && __GNUC__ == 10
```

Comment:
make a comment to the issue link, so we can find it why we have added it

Reply:
Done